backend/uclapi/common/decorators.py

```diff
@@ -86,7 +86,8 @@ def throttle_api_call(token, token_type):
         cache_key = token.user.email
         limit = 10000
     elif token_type == 'general-temp':
-        cache_key = token.api_token
+        # token as a key is already used.
+        cache_key = token + "-key"
```

Comment:
```suggestion
        cache_key = token + "-usage"
```

Reply:
It makes sense to name this key appropriately since it tracks how much the temporary token has been used.